pickee_api/views.py

```diff
@@ -53,11 +53,7 @@ def user_signup(request):
     if request.method == 'POST':
         form = PickeeUserCreationForm(request.POST)
         if form.is_valid():
-            form.save()
-            email = form.cleaned_data['email']
```

Comment:
Are you sure this is not removing our e-mail validations?

Reply:
These two validations still work:
Email address: User with this Email address already exists.
Email address: Enter a valid email address.